CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
 - Modify `Signer`'s `signTransaction` method to return an array of signed bytes rather than a string based signature.
+- Switched the library over to using gRPC-Web instead of the Node.js version of gRPC. The goal of this change is to enable the library to be usable on the web and Node.js without having two separate codebases or build strategies
```

Comment:
Is switching to `grpc-web` a breaking change for a user of `xpring-common-js`?

If so, why?

Reply:
Technically it is not a breaking change for Xpring-Common-JS so I moved it to the added column. Sorry for the confusion.